.github/workflows/integrate-and-deploy.yml

```diff
@@ -41,7 +48,9 @@ jobs:
         id: image
         run: |
           echo ::set-output name=NAMESPACE::sillsdev/web-languageforge
-          echo ::set-output name=TAG::${{ inputs.image-tag }}
+          echo ::set-output name=TAG_APP::${{ inputs.image-tag-app }}
+          echo ::set-output name=TAG_PROXY::${{ inputs.image-tag-proxy }}
+          echo ::set-output name=TAG_NEXT_APP::${{ inputs.image-tag-next-app }}
```

Comment:
yeah, I like this.  I appreciate you trying to do it for me in the suggestion but your namespace is in the wrong place so I'll just push a commit with these changes.

Reply:
well, this may not be as straight-forward as I was thinking, it might require looking at a different location to make those things available but we also need to be mindful of the fact that those are incoming workflow args...maybe we should pair on this one.

Reply:
I defer to your experience and that you tried to make this work.  satisfied for now.